portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/TransferServiceProviderRegistry.java

```diff
@@ -1,15 +1,28 @@
 package org.dataportabilityproject.spi.transfer.provider;
 
+import org.dataportabilityproject.types.transfer.PortableType;
+import org.dataportabilityproject.types.transfer.auth.AuthData;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+
 /**
  * Manages {@link TransferServiceProvider}s registered in the system.
```

Comment:
Update comment

Reply:
Done. Since we still want to keep around the TransferServiceProvider I kept this comment as is. I think we'll still want the implementation of this Registry to keep track of the different TransferServiceProvider, keeping around for now and we can always delete/repurpose if needed.